SMLHelper/Crafting/RecipeData.cs

```diff
@@ -0,0 +1,104 @@
+﻿#if BELOWZERO
+#pragma warning disable IDE1006 // Naming Styles - Ignored for backwards compatibility
+namespace SMLHelper.V2.Crafting
+{
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// A class that fully describes a recipe for a <see cref="TechType"/> identified item.
+    /// </summary>
+    /// <seealso cref="TechData" />
+    public class RecipeData
```

Comment:
Yes ..... BUT  I can see no advantage of doing so as that would just end up with 3 places that hold the techdata instead of just 2. Also Subnautica is said to be DONE with changes except for unity updates and the like but they could still change BelowZero's Techdata again at any update and then we would have to split them up again.

Reply:
In this one instance due to uncertain future changes to Below Zero I do believe it should remain seperate "for now". After BZ full release if they have not muxed it about more then it should not be an issue to do it but again that would just make 3 places instead of 2 for the handling of techdata.

Reply:
If both of those inherit from an abstract class we can have one set of methods for that abstract class, getting rid of most file separation clutter.

Reply:
Well except that both games still handle the games side completely differently.